File: elias/Problema0128/problema0128.java
import java.util.Scanner;

public class problema0128 {

    public static void main(String[] args) {
    Scanner input = new Scanner (System.in);
    float saldoMedio, credito;
    System.out.println("Digite seu saldo médio do último ano:");
    saldoMedio = Float.valueOf(input.nextLine());
        if(saldoMedio < 0){
            System.out.println("Valor inválido, tente novamente!");
        }
        else if(saldoMedio >= 0 && saldoMedio <= 500){
            System.out.println("O seu saldo médio é "+saldoMedio+" e você não possui nenhum crédito.");
        }
        else if(saldoMedio >= 501 && saldoMedio <= 1000){
            credito = saldoMedio * 30 / 100;
            System.out.println("O seu saldo médio é "+saldoMedio+" e você possui um crédito de: "+credito);
            } 
        else if(saldoMedio >= 1001 && saldoMedio <= 3000){
            credito = saldoMedio * 40 / 100;
            System.out.println("O seu saldo médio é "+saldoMedio+" e você possui um crédito de: "+credito);
        }   
        else if(saldoMedio > 3000){
            credito = saldoMedio * 50 / 100;
            System.out.println("O seu saldo médio é "+saldoMedio+" e você possui um crédito de: "+credito);
        }
    }
}

File: elias/Problema0128/problema0128.py
saldoMedio = float(input("Digite seu saldo médio do último ano:"))
if saldoMedio < 0:
    print("Valor inválido, tente novamente")
elif saldoMedio >= 0 and saldoMedio <= 500:
    print("O seu saldo médio é ",saldoMedio," e você não possui nenhum crédito.")
elif saldoMedio >= 501 and saldoMedio <= 1000:
    credito = saldoMedio * 30 / 100
    print("O seu saldo médio é ",saldoMedio," e você possui um crédito de: ",credito)
elif saldoMedio >= 1001 and saldoMedio <= 3000:
    credito = saldoMedio * 40 / 100
    print("O seu saldo médio é ",saldoMedio," e você possui um crédito de: ",credito)   
elif saldoMedio > 3000:
    credito = saldoMedio * 50 / 100
    print("O seu saldo médio é ",saldoMedio," e você possui um crédito de: ",credito)